components/signup/signup_controller/signup_controller.test.jsx

```diff
@@ -99,6 +101,42 @@ describe('components/SignupController', () => {
         expect(browserHistory.push).toHaveBeenCalledWith(`/defaultTeam/channels/${Constants.DEFAULT_CHANNEL}`);
     });
 
+    test('should redirect user to the restricted screen when workspace out of free seats', async () => {
```

Comment:
Could we add a test here that if it's cloud we won't call `getSubscriptionStats`?

Reply:
done!